Add exercises

fingolfin opened the pull request:
This is just a placedholder. We can all push to this branch,

fingolfin committed:
Add exercise stubs

thofma committed:
add some number theory & algebra exercises

thofma commented:
Ich habe mal ein paar "Oscar benutzen" Übungen für Zahlentheorie & Kommutative Algebra hinzugefügt. Preview ist [hier](https://www.thofma.com/oscar-website/meetings/2021-09/exercises/). Das sind keine Implementationssachen, sondern nur "Oscar kennen lernern" Übungen. Ein paar muss ich selbst noch einmal nachrechnen ...

Was denkst du @fieker? Noch ein anderes Themengebiet anschneiden?

fieker commented:
in 5.1: Determinantal ist die mathematik falsch formatiert, in 6.3 ist ein "I" zuviel

Ich finde, das ist eni guter Anfang. Fuer "haertere" Bsp/ Ideen koennen wir ja auch die Leipzig Bsp weiter nutzen...

thofma committed:
moar

thofma committed:
Add more exercises

thofma commented:
OK, I pushed more, preview [here](https://www.thofma.com/oscar-website/meetings/2021-09/exercises/)

 I will stop with adding exercises now. We now have
- 5 julia exercises, different levels (@tthsqe12 is adding more julia exercises)
- 8 exercises for *using* Oscar which involve number theory and a bit of commutative algebra/geometry.
- 4 exercises where one is supposed to *implement* something, also related to number theory and algebra.

They all need a bit of cleanup and I will check that they actually work later.

There is also the question of how to organize the "using Oscar" and "implementing something using Oscar".

ThomasBreuer committed:
added exercises related to Eamonn's talks

tthsqe12 committed:
add more julia exercises

ThomasBreuer committed:
one more exercise (from the list in the HackMD)

ThomasBreuer committed:
one more sub-exercise

thofma committed:
clean up exercises

ThomasBreuer committed:
two more exercises about groups

fingolfin committed:
Update exercises

ThomasBreuer committed:
adjusted the exercises page

- to the changed location of the repository `Summerschool21Exercises`,
- and added some terms from the talk to an exercise, as a hint

thofma committed:
Some changes

thofma committed:
adjust one exercise

fingolfin committed:
Some tweaks
